### ISSUE TITLE
StackdriverExporter fails to emit span from GKE container

### ISSUE THREAD

**Issue by Amey-D** [census-instrumentation/opencensus-python#409]
When using `opencensus-python==0.1.8` in a Python Flask app running as a GKE container, I see the following stacktrace:

```
2018-11-26 09:04:22,779 ERROR background_thread.py:112: StackdriverExporter failed to emit spans.Dropping 1 spans from queue.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/transports/background_thread.py", line 106, in _thread_main
    self.exporter.emit(span_datas)
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/stackdriver_exporter.py", line 224, in emit
    stackdriver_spans = self.translate_to_stackdriver(trace)
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/stackdriver_exporter.py", line 248, in translate_to_stackdriver
    set_attributes(trace)
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/stackdriver_exporter.py", line 76, in set_attributes
    set_monitored_resource_attributes(span)
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/stackdriver_exporter.py", line 91, in set_monitored_resource_attributes
    'project_id')
  File "/usr/local/lib/python3.6/site-packages/opencensus/trace/exporters/stackdriver_exporter.py", line 136, in set_attribute_label
    label_value_prefix + resource_labels[attribute_key]
TypeError: must be str, not bytes
```

Note that the same application configuration setup used to work with `opencensus-python==0.1.4`.  I had to upgrade the package version in order to fix memory leaks in older versions (#334). So this clearly is a regression in recent versions of this package.

**Comment by Amey-D** [comment]
+cc @c24t since you helped investigate #334, I thought this might also interest you.

**Comment by c24t** [comment]
Hey @Amey-D, unfortunately that's a bug in `0.1.8`, fixed on master by #365. I've got an open PR to create a branch for `0.1.8.x` releases, once it's merged (and the new distribution uploaded to PyPI) you should be able to install `0.1.8.1`, which includes the fix.

We should also be able to cut a new release from master soon, details in #403.

**Comment by c24t** [comment]
Closing this as a duplicate of #381 since it's fixed on master, but we need to get this fix into a release soon.
